sentry.client.config.js

```diff
@@ -19,9 +32,12 @@ Sentry.init({
   // don't record replays for generic sessions
   replaysSessionSampleRate: 0,
   // record replays for errors
-  replaysOnErrorSampleRate: 1.0,
+  replaysOnErrorSampleRate: 0.1,
   integrations: [new Sentry.Replay()],
 
   ignoreErrors: sentryOptions.ignoreErrors,
   denyUrls: sentryOptions.denyUrls,
+  // only allow thirdweb.com domains
+  allowUrls: ["https://thirdweb.com"],
```

Comment:
this may be too strict

Reply:
Don't we have to use a regex here? or just "thirdweb.com"

Reply:
think we should allow using regex to support preview envs  + staging too. lets allow anything ending with thirdweb.com

Reply:
I don't think we need to report errors from staging - but I think `https://thirdweb.com` will just report errors on home page and not other pages